lib/internal/main/worker_thread.js

```diff
@@ -147,7 +151,10 @@ port.on('message', (message) => {
           `(eval = ${eval}) at cwd = ${process.cwd()}`);
     port.unref();
     port.postMessage({ type: UP_AND_RUNNING });
-    if (doEval) {
+    if (internal) {
+      debug(`WHTY ${publicPort}`);
```

Comment:
Please use readable debug messages.

Reply:
This looks like a temporary message, removing for now.